TypeError: Cannot read property 'split' of undefined

Running image actions on an existing repository, I received this error: 

```
/usr/local/src/image-actions/src/constants.js:15
const IGNORE_PATHS = process.env["INPUT_IGNOREPATHS"].split(",") || ["node_modules/**"];
                                                      ^

TypeError: Cannot read property 'split' of undefined
    at Object.<anonymous> (/usr/local/src/image-actions/src/constants.js:15:55)
    at Module._compile (internal/modules/cjs/loader.js:816:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:827:10)
    at Module.load (internal/modules/cjs/loader.js:685:32)
    at Function.Module._load (internal/modules/cjs/loader.js:620:12)
    at Module.require (internal/modules/cjs/loader.js:723:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.<anonymous> (/usr/local/src/image-actions/entrypoint.js:3:45)
    at Module._compile (internal/modules/cjs/loader.js:816:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:827:10)
##[error]Docker run failed with exit code 1
```

This is due to the change in #29

--- a/src/constants.js
+++ b/src/constants.js
@@ -1,6 +1,7 @@
 const path = require("path");
 
-const GITHUB_TOKEN = process.env["INPUT_GITHUBTOKEN"] || process.env["GITHUB_TOKEN"];
+const GITHUB_TOKEN =
+  process.env["INPUT_GITHUBTOKEN"] || process.env["GITHUB_TOKEN"];
 const GITHUB_EVENT_NAME = process.env["GITHUB_EVENT_NAME"];
 const GITHUB_EVENT_PATH = process.env["GITHUB_EVENT_PATH"];
 const GITHUB_SHA = process.env["GITHUB_SHA"];
@@ -12,7 +13,9 @@ const REPO_DIRECTORY = process.env["GITHUB_WORKSPACE"];
 const JPEG_QUALITY = parseInt(process.env["INPUT_JPEGQUALITY"]) || 80;
 const PNG_QUALITY = parseInt(process.env["INPUT_PNGQUALITY"]) || 80;
 const WEBP_QUALITY = parseInt(process.env["INPUT_WEBPQUALITY"]) || 80;
-const IGNORE_PATHS = process.env["INPUT_IGNOREPATHS"].split(",") || ["node_modules/**"];
+const IGNORE_PATHS = process.env["INPUT_IGNOREPATHS"]
+  ? process.env["INPUT_IGNOREPATHS"].split(",")
+  : ["node_modules/**"];
 
 const COMMITTER = {
   name: "Calibre",
@@ -22,7 +25,7 @@ const COMMITTER = {
 if (!REPO_DIRECTORY) {
   console.log("::error:: There is no GITHUB_WORKSPACE environment variable");
   process.exit(1);
-};
+}
 
 const CONFIG_PATH = path.join(
   REPO_DIRECTORY,

